src/t_stream.c

```diff
@@ -2727,23 +2740,19 @@ void xautoclaimCommand(client *c) {
         return;
     }
 
-    if (count == 0) {
-        addReplyNullArray(c);
-        return;
-    }
-
     /* Do the actual claiming. */
     streamConsumer *consumer = NULL;
     long long attempts = count*10;
 
     addReplyArrayLen(c, 2);
+    void *endidptr = addReplyDeferredLen(c);
```

Comment:
Should we rename addReplyDeferredLen? 
No longer about len. 
I guess not, will mess up the code base too much